routing/router.go

```diff
@@ -1692,8 +1692,13 @@ func (r *ChannelRouter) applyChannelUpdate(msg *lnwire.ChannelUpdate) error {
 		return nil
 	}
 
-	err := r.UpdateEdge(&channeldb.ChannelEdgePolicy{
-		Signature:                 msg.Signature,
+	sig, err := msg.Signature.ToSignature()
+	if err != nil {
+		return err
+	}
+
+	err = r.UpdateEdge(&channeldb.ChannelEdgePolicy{
+		SigBytes:                  sig.Serialize(),
```

Comment:
Here we can do `msg.Signature.ToSignatureBytes`

Reply:
Fixed